Description: Disabling YUI library SWF components
 Disabled a removed SWF file, that has been removed
 to conform the DFSG.
Author: Markus Frosch <markus@lazyfrosch.de>
Last-Update: 2013-05-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/app/modules/AppKit/lib/js/AppKit.js
+++ b/app/modules/AppKit/lib/js/AppKit.js
@@ -54,7 +54,17 @@
                 var me = AppKit;
 
                 Ext.BLANK_IMAGE_URL = me.c.path + '/images/ajax/s.gif';
-                Ext.chart.Chart.CHART_URL = me.c.path + '/js/ext3/resources/charts.swf';
+                // BEGIN Debian - see /usr/share/doc/icinga-web/README.Debian
+                // disabled external Flash components
+                Ext.chart.Chart.CHART_URL = null;
+                Ext.FlashComponent.EXPRESS_INSTALL_URL = null;
+                // enable the following for local stored files
+                //Ext.chart.Chart.CHART_URL = me.c.path + '/js/ext3/resources/charts.swf';
+                //Ext.FlashComponent.EXPRESS_INSTALL_URL = me.c.path + '/js/ext3/resources/expressinstall.swf';
+                // enable the following for remote stored resources
+                //Ext.chart.Chart.CHART_URL = 'http:/' + '/yui.yahooapis.com/2.8.2/build/charts/assets/charts.swf';
+                //Ext.FlashComponent.EXPRESS_INSTALL_URL = 'http:/' + '/swfobject.googlecode.com/svn/trunk/swfobject/expressInstall.swf';
+                // END Debian
 
                 Ext.QuickTips.init();
                 growlStack();
@@ -270,4 +280,4 @@
     }()));
 })();
 
-Ext.ns('AppKit.lib', 'AppKit.util');
\ No newline at end of file
+Ext.ns('AppKit.lib', 'AppKit.util');
